VulnerableApplication/Global.asax.cs

```diff
@@ -10,12 +10,19 @@ namespace VulnerableApplication
 {
     public class MvcApplication : System.Web.HttpApplication
     {
+        public string Password = "abcdef";
+    
```

Comment:
[VULNERABILITY] Make this field 'private' and encapsulate it in a 'public' property.